ajax/libs/redux-persist/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "redux-persist",
+  "version": "4.0.0-alpha6",
+  "description": "persist and rehydrate redux stores",
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "jsnext:main": "es/index.js",
+  "ava": {
+    "require": [
+      "babel-register"
+    ],
+    "babel": "inherit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/rt2zz/redux-persist.git"
+  },
+  "homepage": "https://github.com/rt2zz/redux-persist",
+  "keywords": [
+    "redux",
+    "redux-middleware",
+    "localstorage",
+    "redux-persist",
+    "redux-storage",
+    "redux-rehydrate"
+  ],
+  "author": "rt2zz <zack@root-two.com>",
+  "license": "MIT",
+  "dependencies": {
+    "json-stringify-safe": "^5.0.1",
+    "lodash": "^4.11.1",
+    "lodash-es": "^4.16.0"
+  },
+  "files": [
+    "es",
+    "dist",
+    "lib",
+    "src",
+    "constants.js",
+    "storages.js",
+    "redux-persist.js.flow"
+  ],
+  "standard": {
+    "parser": "babel-eslint"
+  },
+  "npmName": "redux-persist",
+  "npmFileMap": [
+    {
+      "basePath": "/src",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "/autoRehydrate.js"
```

Comment:
@ian4hu Please point to the main file. It's `"redux-persist.min.js"` in this case.